### ISSUE TITLE
AKS cluster provisioning fails with default options as Kubernetes version is not being passed by UI

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#16668]
Rancher version: v2.1.2-rc13

Steps:
1. Create an AKS cluster with default options set
2. Cluster creation fails with the below error as Kubernetes version is not passed(NULL)
```
Containerservice.ManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidParameter" Message="The value of parameter orchestratorProfile.OrchestratorVersion is invalid." Target="orchestratorProfile.OrchestratorVersion"    
```

```
{dockerRootDir: "/var/lib/docker", enableNetworkPolicy: false, type: "cluster",…}
azureKubernetesServiceConfig: {count: 3, osDiskSizeGb: 100, agentPoolName: "rancher", type: "azureKubernetesServiceConfig",…}
adminUsername: "azureuser"
agentPoolName: "rancher"
agentVmSize: "Standard_D2_v2"
clientId: "12345"
clientSecret: "abcd="
count: 3
dnsServiceIp: null
dockerBridgeCidr: null
kubernetesVersion: null
location: "eastus"
osDiskSizeGb: 100
resourceGroup: "soumya-newrg"
serviceCidr: null
sshPublicKeyContents: "ssh-rsa abcdef"
subnet: null
subscriptionId: "abcdef"
tenantId: "ghijkl"
type: "mnopq"
virtualNetwork: null
virtualNetworkResourceGroup: null
dockerRootDir: "/var/lib/docker"
enableNetworkPolicy: false
name: "testaks"
type: "cluster"

```

**Comment by westlywright** [comment]
Available in UI Tag: `2.1.16`

**Comment by tfiduccia** [comment]
Version - 2.1.2-rc14
Verified fixed and cluster created.
